### ISSUE TITLE
Invalid parsing of connection url

### ISSUE THREAD

**Issue by piotr-placzek** [jorgerojas26/lazysql#147]
I just download and run latest release binary file.
I'm trying to connect with my base using url:
`mysql://user:pass@localhost:3008/dev`

I've got message:
```
parse mysql://user:pass@localhost:3008/dev": invalid port "pass" after host
```

It's even can't be saved.

**Comment by jorgerojas26** [comment]
Please be sure your password does not contain special characters, if it does, URL parse it.

**Comment by piotr-placzek** [comment]
You are right. Password contains small and big letters, numbers and special characters. I think it is standard policy today.

**Comment by jorgerojas26** [comment]
Glad it worked.

For the "Connection refused", looks like the port is not correct. Also, check credentials.
